crypto/pkcs7/pkcs7.c

```diff
@@ -1529,6 +1529,9 @@ static int pkcs7_signature_verify(BIO *in_bio, PKCS7 *p7, PKCS7_SIGNER_INFO *si,
   ret = 1;
 
 out:
+#if defined(BORINGSSL_UNSAFE_FUZZER_MODE)
```

Comment:
here and elsewhere, why does this always need to succeed when fuzzing? suggest adding a comment explaining.

Reply:
Same question here, I was under the impression that functions didn't necessarily have to succeed during fuzz testing? We just have to be sure that we're handling all cases properly and that there isn't any undefined behavior.
`fuzz/read_pem.cc` might be a good example of us testing against `PEM_read_bio` without the extra knob.

Reply:
We want `pkcs7_signature_verify` to succeed here during fuzz testing so that the for loop in `PKCS7_verify` completes all of its iterations.  I'll add a comment relating to that.